enhancements/run-bundle.md

```diff
@@ -0,0 +1,263 @@
+---
+title: run-bundle
+authors:
+  - "@jmrodri"
+reviewers:
+  - "@joelanford"
+  - "@estroz"
+  - "@kevinrizza"
+  - "@ecordell"
+  - "@shawn-hurley"
+approvers:
+  - "@joelanford"
+  - "@estroz"
+  - "@ecordell"
+  - "@shawn-hurley"
+creation-date: 2020-05-07
+last-updated: 2020-05-07
+status: implementable
+see-also:
+replaces:
+superseded-by:
+
+---
+
+# Run Bundle
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+1. We currently check OLM installation status with `operator-sdk olm status`.
+   Can we change that to using the discovery API?
+   * Based on discussions, using the discovery API is the way to go.
+
+2. How do we handle --mode= flag on registry add
+   * should we ALWAYS default to --mode=semver?
+     * this has less testing and isnot yet the default. We should stick with
```

Comment:
I would personally like to suggest that `--mode=semver` should be the default. It isn't mainly for backwards compatibility reasons rather than because it's an in progress feature at this point.

Reply:
What about semver-skippatch? I imagine a some users will want to pick between modes for sdk as well.

Reply:
Semver is a logical choice, and users can choose semver-skippatch explicitly if they'd like. We will document different mode behavior in relation to `run bundle`.

Reply:
Based on all our discussions.
* When adding a bundle to an empty index, default to **semver**
* When adding a bundle to a provided index, default to **replaces**
* Should the add mode be configurable on the CLI? **optional flag**; not in an example right now